src/app/components/rss/rss-feed-all/rss-feed-all.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Input } from '@angular/core';
+import { RssFeedService } from '../../../services/rss.service';
+
+@Component({
+  selector: 'app-rss-feed-all',
+  templateUrl: './rss-feed-all.component.html',
+  styleUrls: ['./rss-feed-all.component.scss'],
+})
+export class RssFeedAllComponent {
+  @Input() rssLink!: string[];
+  rssDataItems?: any[] = [];
+
+  constructor(private rssFeedService: RssFeedService) {}
+
+  ngOnInit(): void {
+    this.getRssData();
+  }
+
+  getRssData(): void {
```

Comment:
Good job! It is a bit imperative however, if you need to use this subscribing method in another file you'll have to duplicate code.
If you don't need it anywhere else, keep it like that, else transfer all the data transformation and sorting into the service in order to just give the method to your component

Reply:
the two functions of title In item and sort asc can be reused in the display of rss feeds specific to a feed or for later when filtering. And I modified the api link to also provide filtering.